_data/authors.yml

```diff
@@ -1,16 +1,16 @@
 # /_data/authors.yml
 
 Your_Name:
```

Comment:
Chenge this variable to you name. Reference it in the author section on your posts.

Reply:
Do not delete the Sample that was already present there! Rather, put your entry below that.